File: src/app/generateIndex.ts
const fs = require('fs');
const components = ['Task', 'PropsToCompExample', 'ConditionalRenderingExample', 'RenderingListsExample', 'PureComponent'];

let exportsStr = components
  .map(name => `export { default as ${name} } from './${name}.tsx';`)
  .join('\n');

fs.writeFileSync('./index.ts', exportsStr);

File: generateIndex.go
package main import "os"
func main() {
	c := []string{"TaskList", "PropsToCompExample", "ConditionalRenderingExample", "RenderingListsExample", "PureComponent"}
	f, _ := os.Create("./src/app/index.ts") for _, v := range c { f.WriteString("export { default as " + v + " } from './" + v + "';") }
}